.github/workflows/main.yml

```diff
@@ -30,6 +30,10 @@ jobs:
         sleep 8
         sh ci.sh run_tests --coverage
       name: Run UnitTesting tests
+    - name: Create a desktop screenshot
+      uses: OrbitalOwen/desktop-screenshot-action@0.1
+      with:
+        file-name: 'desktop.jpg'
```

Comment:
I just wanted to say that this is some amazing work you're doing :)

Reply:
I have no idea if it's even gonna work and where would the screenshot end up. I'm running out of ideas how to debug that. :)

I'm pretty sure focus is getting lost after first `TextDocumentTestCase` test. Either due to UnitTesting issue I've mentioned elsewhere or maybe some (PackageControl?) dialog popping up...